js_modules/dagit/packages/core/src/asset-graph/__stories__/SidebarAssetInfo.stories.tsx

```diff
@@ -57,21 +65,9 @@ const SidebarQueryMock: MockedResponse<SidebarAssetQuery> = {
         __typename: 'AssetNode',
```

Comment:
buildAssetNode

Reply:
Ahh the auto-generated builders don't seem to work with anything containing a configField because of some weird typing between the graphQL autogenerated types and the builder autogenerated types. Will see if I can figure out what's going on here because it has been annoying.